tests/acceptance/breadcrumbs/scan-details-test.js

```diff
@@ -0,0 +1,450 @@
+import { visit, click, findAll } from '@ember/test-helpers';
```

Comment:
Some test cases are missing here. 

1. The file compare pages. Including project All project file page to the file compare page and the file choose page to the file compare page. Also account for the vulnerability compare page from the file compare page.

2. From the scan action page (DAST, SAST, etc) --> Vulnerability Details Page